guardian/cdk-cli
# Add migrated prop to GuStackProps on migrate

## What does this change?

This PR adds the `migratedFromCloudFormation` prop to stacks created by running the `migrate` command. This prop was introduce to the `@guardian/cdk` library in https://github.com/guardian/cdk/pull/135 and means that certain constructs will automatically override their id.

## How to test

Run the `migrate` command and see the `migratedFromCloudFormation` prop in the app file.

## How can we measure success?

Migrated stacks set the `migratedFromCloudFormation` to enable specific behaviour by default
